canonical-web-and-design/snapcraft.io
# Simple notification component and show error for releases

## QA

- Pull the branch
- `./run`
- Visit https://snapcraft.io/charm/releases
- See an error message
